docsrc/tutorials/deploy_torch_tensorrt_to_triton.rst

```diff
@@ -0,0 +1,212 @@
+Deploying a Torch-TensorRT model (to Triton)
+============================================
+
+Optimization and deployment go hand in hand in a discussion about Machine 
+Learning infrastructure. For a Torch-TensorRT user, network level optimzation 
+to get the maximum performance would already be an area of expertize. 
+
+However, serving this optimized model comes with it's own set of considerations
+and challenges like: building an infrastructure to support concorrent model
+executions, supporting clients over HTTP or gRPC and more.
+
+The `Triton Inference Server <https://github.com/triton-inference-server/server>`__ 
+solves the aforementioned and more. Let's discuss step-by-step, the process of
+optimizing a model with Torch-TensorRT, deploying it on Triton Inference
+Server, and building a client to query the model. 
+
+Step 1: Optimize your model with Torch-TensorRT
+-----------------------------------------------
+
+Most Torch-TensorRT users will be familiar with this step. For the purpose of
+this demoonstration, we will be using a ResNet50 model from Torchhub.
+
+Let’s first pull the NGC PyTorch Docker container. You may need to create 
+an account and get the API key from `here <https://ngc.nvidia.com/setup/>`__. 
+Sign up and login with your key (follow the instructions
+`here <https://ngc.nvidia.com/setup/api-key>`__ after signing up).
+
+::
+
+   # <xx.xx> is the yy:mm for the publishing tag for NVIDIA's Pytorch 
+   # container; eg. 22.04
+
+   docker run -it --gpus all -v /path/to/folder:/resnet50_eg nvcr.io/nvidia/pytorch:<xx.xx>-py3
```

Comment:
`-v /path/to/folder:/resnet50_eg`  - is this required here ?

Reply:
If so, please elaborate in the README as to which folder we should mount to docker

Reply:
`path/to/local/folder/to/copy/model:/resnet50_eg` is this required here ? 
Like torch.hub will automatically download the RN50 model right  as per `model = torch.hub.load('pytorch/vision:v0.10.0', 'resnet50', pretrained=True).eval().to("cuda")` ?

Reply:
Also, `resnet50_eg` it is not being used anywhere in the instructions. We don't need this I think

Reply:
Is this resolved ?